src/Workload/Microsoft.NET.Sdk.Maui/WorkloadManifest.in.targets

```diff
@@ -1,4 +1,6 @@
 <Project>
+
+  <!-- Known framework references -->
   <PropertyGroup>
```

Comment:
It looks like this is moving most or all of what was in BundledVersions.targets into WorkloadManifest.targets.

We should move *only* the `KnownFrameworkReference` items, not any of the other logic.  We shouldn't set any properties in the workload manifest file, and the KnownFrameworkReferences should be modified to hard-code the versions (to `**FromWorkload**` instead of using the Maui properties.

Reply:
Why do you need the `RuntimePackRuntimeIdentifiers` to differ at all?  Can you just list all of the RuntimeIdentifiers where there is a runtime pack available, whether it's applicable to the current project or not?  That's what we do for our runtime packs, for example:

```
RuntimePackRuntimeIdentifiers="linux-arm;linux-arm64;linux-musl-arm64;linux-musl-x64;linux-x64;osx-x64;rhel.6-x64;tizen.4.0.0-armel;tizen.5.0.0-armel;win-arm;win-arm64;win-x64;win-x86;linux-musl-arm;osx-arm64;maccatalyst-x64;maccatalyst-arm64;linux-s390x"
```

The idea with avoiding properties is to avoid polluting non-Maui projects with extra properties, and to avoid risk that the logic in the workload manifest targets would break a non-Maui project.  However, thinking about this more, if you want to preserve the option to override these versions, then I think it would be OK to have something like this in the workload manifest targets:

```xml
<ItemGroup>
  <MauiVersion Condition=" '$(MauiVersion)' == '' ">**FromWorkload**</MauiVersion>
</ItemGroup>
```

Reply:
Right now there is a different pack per platform, like `Microsoft.Maui.Core.Ref.android` and `Microsoft.Maui.Core.Runtime.android`, and so `%(RuntimePackRuntimeIdentifiers)` only lists the one platform.

Trying something like this, but I don't think you can batch during evaluation:
```xml
<_MauiRuntimeIdentifiers Include="any;android;ios;maccatalyst;win" />
<KnownFrameworkReference
    Condition=" '$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' "
    Include="Microsoft.Maui.Core"
    TargetingPackName="Microsoft.Maui.Core.Ref.%(_MauiRuntimeIdentifiers.Identity)"
    RuntimePackRuntimeIdentifiers="%(_MauiRuntimeIdentifiers.Identity)"
```

Reply:
Are there different contents of the targeting pack for the different platforms?  Normally there would be just one targeting pack, and then separate runtime packs for each RID.  The APIs / interface in the targeting pack would be the same, just the implementations would be different.

Reply:
There are platform-specific APIs, one example is:

https://github.com/dotnet/maui/blob/main/src/Core/src/Platform/Android/DrawableExtensions.cs

If you're writing your own MAUI "handler" specific for Android, for example, you might use this. This is just one example, though.